app/src/main/java/com/protify/protifyapp/features/calendar/EventBreakdown.kt

```diff
@@ -0,0 +1,373 @@
+    )
+
+    private fun createListEvent(): List<Event> {
+        // placeholder until event logic is implemented
+        return listOf(
+            Event().apply {
+                title = "Event 1"
+                startTime = "07:00"
+                endTime = "08:30"
+                description = "Description for Event 1"
+                location = "Location for Event 1"
+                attendees = listOf()
+                id = "1"
+            },
+            Event().apply {
+                title = "Event 2"
+                startTime = "07:00"
+                endTime = "10:00"
+                description = "Description for Event 2"
+                location = "Location for Event 2"
+                attendees = listOf()
+                id = "2"
+            },
+            Event().apply {
+                title = "Event 3"
+                startTime = "12:00"
+                endTime = "13:00"
+                description = "Description for Event 3"
+                location = "Location for Event 3"
+                attendees = listOf()
+                id = "3"
+            },
+            Event().apply {
+                title = "Event 4"
+                startTime = "14:00"
+                endTime = "15:30"
+                description = "Description for Event 4"
+                location = "Location for Event 4"
+                attendees = listOf()
+                id = "4"
+            }
+        )
+    }
+
+    @Composable
+    fun DailySchedule(
+        scale: Double = 1.0,
+        eventList: List<Event> = listOf(),
+        uid: String = "",
+        day: String = "",
+        month: String = "",
+        year: String = "",
+        showTimes: Boolean = true
+    ) {
+        val context: Context = LocalContext.current
+        val timeSlots = if (eventList.isEmpty()) createListTimeSlot() else createListTimeSlot(eventList)
+        val scaledValue = scale.toInt()
+        timeSlots.maxOfOrNull { it.height } ?: 0f
+        val scrollState = rememberScrollState()
+        val layers = timeSlots.map { it.layer }.distinct()
+
+        val startTime = if (eventList.isNotEmpty())
+            convertTimeToFloat(eventList[0].startTime) else 9f // default 0900
+        val startPosition = startTime * 60 * scaledValue
+        LaunchedEffect(key1 = true) {
+            scrollState.scrollTo(startPosition.toInt())
+        }
+
+        Box(modifier = Modifier.fillMaxSize()) {
+            TimeGrid(scaledValue, scrollState, showTimes)
+            for (layer in layers) {
+                TimeSlotLayer(timeSlots, scaledValue, layer, context, uid, day, month, year, scrollState)
+            }
+        }
+    }
+
+    private fun createListTimeSlot(events: List<Event> = createListEvent()): List<TimeSlot> {
+        val timeSlots = mutableListOf<TimeSlot>()
+
+        for (event in events) {
+//            val layer = calculateLayer(event, events)
+//            val layer = events.indexOf(event)
+            val layer = 0
```

Comment:
Removed layers to fix problems where events were rendering on top of each other. This functionality doesn't seem to work properly.